bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -85,8 +85,8 @@ func (r *KubeadmConfigReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Complete(r)
 }
 
-// Reconcile handles KubeadmConfig events
-func (r *KubeadmConfigReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, rerr error) {
+// Reconcile handles KubeadmConfig events.
+func (r *KubeadmConfigReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, rerr error) { //nolint
```

Comment:
Why is nolint added here?

Reply:
Gocyclo complains this function is too big

Reply:
Can we add a comment and possibly an issue to track resolving rather than just excluding linting?

Reply:
https://github.com/kubernetes-sigs/cluster-api/issues/1552